frontend/dockerfile/docs/reference.md

```diff
@@ -1135,33 +1132,22 @@ RUN apt-get update && apt-get install -y ...
 
 ## ADD
 
-ADD has two forms:
+ADD has two forms.
+The latter form is required for paths containing whitespace.
 
 ```dockerfile
-ADD [--chown=<user>:<group>] [--chmod=<perms> [--exclude=<exclude>]... [--checksum=<checksum>] <src>... <dest>
-ADD [--chown=<user>:<group>] [--chmod=<perms> [--exclude=<exclude>]... ["<src>",... "<dest>"]
+ADD [OPTIONS] <src> ... <dest>
+ADD [OPTIONS] ["<src>", ... "<dest>"]
 ```
 
-The latter form is required for paths containing whitespace.
+The available `[OPTIONS]` are:
 
-> **Note**
->
-> The `--chown` and `--chmod` features are only supported on Dockerfiles used to build Linux containers,
-> and doesn't work on Windows containers. Since user and group ownership concepts do
-> not translate between Linux and Windows, the use of `/etc/passwd` and `/etc/group` for
-> translating user and group names to IDs restricts this feature to only be viable
-> for Linux OS-based containers.
-
-> **Note**
->
-> `--chmod` is supported since [Dockerfile 1.3](https://docs.docker.com/build/buildkit/dockerfile-frontend/).
-> Only octal notation is currently supported. Non-octal support is tracked in
-> [moby/buildkit#1951](https://github.com/moby/buildkit/issues/1951).
-
-> **Note**
->
-> The `--exclude` option can be specified multiple times and cause files matching its patterns not to be copied,
-> even if the files paths match the pattern specified in `<src>`. This feature requires the build tag `dfexcludepatterns`.
+- [`--keep-git-dir`](#add---keep-git-dir)
```

Comment:
In this list of flags for ADD/COPY, would it make sense to put "(since vX.X)" behind them so it is clear when they can be used.